PuzzleSlider/src/Engine/GameData.java

```diff
@@ -40,6 +44,10 @@ public String toString() {
 			out.append( Arrays.toString(mTiles[i]));
 			out.append(System.lineSeparator());
 		}
+		out.append("Pocet tahov: " + mMoveCount);
```

Comment:
So, yes.Please, rewrite to english. Thanks. And to the second point, I dont know if it is irrellevant in java, so it is your choice ;)